packages/scandipwa/src/component/WishlistItem/WishlistItem.container.js

```diff
@@ -134,14 +136,21 @@ export class WishlistItemContainer extends PureComponent {
             }
         } = item;
 
-        const configurableVariantIndex = this.getConfigurableVariantIndex(sku, variants);
-        const product = type_id === 'configurable'
-            ? { ...item, configurableVariantIndex }
-            : item;
+        if (type_id === 'configurable') {
+            const configurableVariantIndex = this.getConfigurableVariantIndex(sku, variants);
+
+            if (!configurableVariantIndex) {
+                history.push({ pathname: appendWithStoreCode(item.url) });
+                showNotification('info', __('Please select product options!'));
+                return;
+            }
+
+            item.configurableVariantIndex = configurableVariantIndex;
+        }
 
         this.setState({ isLoading: true });
 
-        return addProductToCart({ product, quantity })
+        addProductToCart({ product: item, quantity })
```

Comment:
Why has this modification been made?
It's no longer possible to do `this.addItemToCart.then(...)`